wazuh/wazuh-dashboard-plugins
# The automatic page refresh is not working in Wazuh Dashboard in version 4.9.0

### Description
The automatic page refresh is not working properly for the Wazuh Dashboard in version 4.9.0. The page is refreshing but it is not updating the changes on the Dashboard. 
![image](https://github.com/user-attachments/assets/2c36fa0b-c85b-4501-b564-101afe1ce7fb)
 
### Issues Resolved
[#25617](https://github.com/wazuh/wazuh/issues/25617)

### Evidence

https://github.com/user-attachments/assets/9246fc29-d12c-421c-8b20-b210f6b08d96


### Test
1. Remove the sample data, if present.
2. Add the sample data to obtain the latest timestamps.
3. Navigate to any application.
4. Set a refresh interval of 5 seconds within the time filter.
5. Observe whether the histogram updates over time.

> [!IMPORTANT]
> 1. Check if the URL updates with the savedQuery when you select an option or clear all (savedQueries)
> 2. Additionally, verify that the AppFilters are being saved correctly in the URL

https://github.com/user-attachments/assets/768486d4-2e5e-42e5-8436-d278b1fcdeb8

### Check List
